feat(components): add knob component (see https://primevue.org/knob/)

wizount opened the pull request:
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

wizount committed:
feat(components): add knob component (see https://primevue.org/knob/)

btea commented:
Thanks for the contribution, but this seems to be the effect of `el-progress`. https://element-plus.org/en-US/component/progress.html#circular-progress-bar

wizount commented:
> Thanks for the contribution, but this seems to be the effect of `el-progress`. https://element-plus.org/en-US/component/progress.html#circular-progress-bar

Thanks for reply.  But the Knob can change the value by mouse click, and the Process can't.

wizount committed:
Merge branch 'dev' into dev

wizount committed:
feat(components): the percentage of el-process can be changed by click
see the effects on https://primevue.org/knob/)
Usages: <el-progress v-model:percentage="processVal" :min="-100" type="circle"></el-progress>

wizount committed:
Merge remote-tracking branch 'origin/dev' into dev
